### PR TITLE
Add Swift docs

### PR DESCRIPTION
**Pull request by victorg1991**
Hey!

I've added all the swift examples in the docs!

Let me know what you think :)

### CODE REVIEW

**Comment by zenorocha** [comment]
That's great! Could you also add instructions on how to include the Swift API Client. Feel free to make changes on this page (http://wedeploy.com/docs/intro/using-the-api-client.html).

**Comment by victorg1991** [comment]
@zenorocha done! :D

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: ui/src/pages/docs/auth/getting-started.md [hunk 1]
```diff
@@ -141,6 +141,10 @@ By using WeDeploy API client you can initialize the authentication service by re
 WeDeploy.auth('http://<serviceID>.<projectID>.wedeploy.me');
 ```
 
+```text/x-swift
+let auth = WeDeploy.auth('http://<serviceID>.<projectID>.wedeploy.me');
```
Comment (zenorocha) [inline ui/src/pages/docs/auth/getting-started.md]: no need for `let`

File: ui/src/pages/docs/auth/getting-started.md [hunk 2]
```diff
@@ -149,6 +153,11 @@ Note that if you are initializing an auth service pointing to the Cloud you shou
 WeDeploy.auth('http://<serviceID>.<projectID>.wedeploy.io');
 ```
 
+```text/x-swift
+let auth = WeDeploy.auth('http://<serviceID>.<projectID>.wedeploy.io');
```
Comment (zenorocha) [inline ui/src/pages/docs/auth/getting-started.md]: no need for `let`

File: ui/src/pages/docs/auth/manage-users.md [hunk 3]
```diff
@@ -47,6 +59,18 @@ if (currentUser) {
...
 
+```text/x-swift
+WeDeploy
+	.auth("<auth-url>")
+    .getCurrentUser()
+    .then { user in
+        // User found.
+    }
+    .catch { error in
+        // User does not exist.
```
Comment (zenorocha) [inline ui/src/pages/docs/auth/manage-users.md]: wrong indentation

File: ui/src/pages/docs/auth/manage-users.md [hunk 4]
```diff
@@ -65,6 +89,18 @@ WeDeploy
...
 
+```text/x-swift
+WeDeploy
+	.auth("<auth-url>")
+    .getUser(id: "userId")
+    .then { user in
+        // User found.
+    }
+    .catch { error in
+        // User does not exist.
```
Comment (zenorocha) [inline ui/src/pages/docs/auth/manage-users.md]: wrong indentation

File: ui/src/pages/docs/auth/sign-in-with-password.md [hunk 5]
```diff
@@ -28,6 +28,19 @@ WeDeploy
...
+let auth = WeDeploy
+	.auth("<auth-url>")
+	.signInWith(username: "user@domain.com", password: "password")
+    .then { auth in
+		// User is signed in.
+    }
+    .catch { err in
+		// User is not signed in.
+    }
+```
```
Comment (zenorocha) [inline ui/src/pages/docs/auth/sign-in-with-password.md]: wrong indentation

File: ui/src/pages/docs/data/saving-data.md [hunk 6]
```diff
@@ -76,6 +90,26 @@ WeDeploy
...
+	.create(resource: "movies", object: [
+	[
+  		"title" : "Star Wars III",
+  		"year" : 2005,
+  		"ratings" : 8.0
+	],
+	[
+     "title" : "Star Wars II",
+     "year" : 2002,
+     "ratings" : 8.6
```
Comment (zenorocha) [inline ui/src/pages/docs/data/saving-data.md]: wrong indentation

**Comment by zenorocha** [comment]
I left some comments to exemplify.

**Comment by victorg1991** [comment]
Fixed!

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/wedeploy/wedeploy.com/compare/b04ffd3~1...b04ffd3) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Thank you so much @victorg1991! It's merged and live!

**Comment by zenorocha** [comment]
I noticed just one code snippet that wasn't "translated" to Swift.

Line #320: https://github.com/wedeploy/wedeploy.com/blob/master/ui/src/pages/docs/data/searching-data.md

**Pull request closed without merging**